praw/reddit.py

```diff
@@ -130,6 +132,7 @@ def __init__(self, site_name=None, **config_settings):
                                   'must be set to None via a keyword arugment '
                                   'to the `Reddit` class constructor.')
 
+        self.requestor = requestor
```

Comment:
Done, passing both requestor_class and requestor_kwargs, no use keeping them around.